src/retasc/models/rule.py

```
@@ -63,7 +65,16 @@ def update_state(self, context) -> ReleaseRuleState:
             context.template.params["state"] = rule_state
 
             with context.report.section(prereq.section_name(context)):
-                state = prereq.update_state(context)
+                try:
+                    state = prereq.update_state(context)
+                except (
+                    PrerequisiteUpdateStateError,
+                    jinja2.exceptions.TemplateError,
+                ) as e:
+                    context.report.set("error", f"❌ {e}")
```

What is the red cross about?

---

Should it be better to call this a "failure" instead of "error" as we continue with a record?

Just minor concern, the PR LGTM.

---

I can't see some approve button?

---

> What is the red cross about? Not all the system can represent the sign correctly

I've added it, so it is easier to spot errors in logs. It may not be rendered with some systems/fonts/non-unicode apps, but I think that most of the time it will be visible.

> Should it be better to call this a "failure" instead of "error" as we continue with a record?

I usually look up just error in logs.

> I can't see some approve button?

You are not in [retasc-owners](https://github.com/orgs/release-engineering/teams/retasc-owners) team. I've invited you.